Neuro/Interactions/InteractionsHandler.cs

```diff
@@ -57,23 +58,42 @@ public void UseTarget(IUsable usable)
                 UseSystemConsole(system);
                 break;
 
-            // case Ladder ladder:
-            //     break;
+            case Ladder ladder:
+                // Check if the player is facing the destination of the ladder because the position of the ladder is below/above the actual ladder
+                if (isPlayerFacingPosition(ladder.Destination.transform.position)) {
+                    PlayerControl.LocalPlayer.UseClosest();
+                }
```

Comment:
Don't trigger the interaction until the player collides with the wall collider and stops moving (you can't check the body velocity directly i don't think because that is overriden, but there might be another unity field that you can check to see if the player is moving), we don't want a random jittering input to snap the player to the ladder from very far away.